src/OpenLoco/Ui/TextInput.cpp

```diff
@@ -166,4 +166,9 @@ namespace OpenLoco::Ui::TextInput
                 }),
             buffer.end());
     }
+
+    uint16_t InputSession::getCharactersLeft()
```

Comment:
Indeed, it looks like this method can be removed. The logic looks incorrect, either way, because the `cursorPosition` reflects the current cursor position (which can be moved), not the string length.